src/ios/GeolocationPlugin.m

```diff
@@ -0,0 +1,276 @@
+//
+//  GeolocationPlugin.m
+//  Geolocation
+//
+//  Created by dev on 2/9/16.
+//
+//
+
+#import "GeolocationPlugin.h"
+
+@implementation GeolocationPlugin
+
+-(void)start:(CDVInvokedUrlCommand *)command
+{
+    coord1 = [command.arguments objectAtIndex:0];   // Get center and radio of region1
```

Comment:
see spec update, input should be a list of coordinates, not just two. maybe to make it more clear:

```
var coords = [coord1, coord2, ...]; // any number of coords
var url = 'http://....'; // server url
start(coords, url);
```